server/Game.js

```diff
@@ -154,21 +158,34 @@ export default class {
     }
   }
 
+  getShuffledCandidates = (candidatesAmount = MAX_CANDIDATES) => {
```

Comment:
I would make this a regular method instead of an arrow function and remove the `candidatesAmount` argument as we're probably never going to call it with anything other than `MAX_CANDIDATES` :-)
```js
getShuffledCandidates() {
```